code/processes/tailreader.q

```diff
@@ -6,6 +6,9 @@ getpartition:@[value;`getpartition;                                        /-fun
 currentpartition:@[value;`currentpartition;getpartition[]]
 basedir:raze (getenv`KDBTAIL),"/tailer",(string .ds.segmentid),"/"         /-define associated tailer base directory
 taildir:`$ basedir,string currentpartition;                                /-define tailDB direction
+tailertypes:`$"tailer_",last "_" vs string .proc.proctype                  /-define tailer to make connection to 
```

Comment:
this seems a bit fragile to me - IMO it might be better to have this config driven? as-is I think it ties users into a fairly specific naming convention for proctypes

also minor nitpick but if there's only going to be one type, variable should be called `tailertype` IMO